src/MetadataUtility/Metadata/SupportFiles/FLLogFile.cs

```diff
@@ -0,0 +1,61 @@
+// <copyright file="FLLogFile.cs" company="QutEcoacoustics">
+// All code in this file and all associated files are the copyright and property of the QUT Ecoacoustics Research Group.
+// </copyright>
+
+namespace MetadataUtility.Metadata.SupportFiles
+{
+    using System.Text.RegularExpressions;
+    using LanguageExt;
+
+    public static class FLLogFile
```

Comment:
We prefer to avoid initialisms. Maybe just add this to a `FrontierLabs` namespace & folder like we do for other file types